src/test/spec/unified_runner/test_runner.rs

```diff
@@ -115,8 +116,19 @@ impl TestRunner {
                     };
                     (id, collection.into())
                 }
-                TestFileEntity::Session(_) => {
-                    panic!("Explicit sessions not implemented");
+                TestFileEntity::Session(session) => {
+                    let id = session.id.clone();
+                    let client = self.get_client(&session.client);
+                    let options: Option<SessionOptions> = session
```

Comment:
Rather than doing this conversion manually, we can update the `Session` struct [here](https://github.com/mongodb/mongo-rust-driver/blob/4bd9f8eb3a56bd8bdb8285438e55497fa1298c0d/src/test/spec/unified_runner/test_file.rs#L208) to use `Option<SessionOptions>` rather than `Option<Document>`. I put `Document` there originally as a placeholder because explicit sessions (and by extension the `SessionOptions` struct) hadn't been implemented yet.